NuRadioMC/test/SignalProp/run_signal_test.sh

```diff
@@ -5,6 +5,7 @@ python3 T02test_analytic_D_T.py
 python3 T04MooresBay.py
 python3 T05unit_test_C0_SP.py
 python3 T06unit_test_C0_mooresbay.py
+python3 T07test_birefringence.py
```

Comment:
also add the execution of all birefringence examples to this script.

Reply:
the examples don't need to be validated, but running them ensures that they keep up to date

Reply:
Some of the examples use radiopropa but the unit tests fail when trying to import it, is there a good way to solve this?

Reply:
We could add s separate test class that installs RadioPropa... but I'm not sure if this is worth it.

Reply:
fixed